docs/gds-supported-platform/resizing-persistent-volumes.md

```diff
@@ -0,0 +1,93 @@
+# Resizing persistent volumes
+
+The volumes you created in the beginning may not be big enough for
+your ongoing needs.
+
+As an example, we're going to double the space on the Concourse Worker
+persistent volumes - from 64GiB to 128GiB. Replace
+`gsp-concourse-worker` with the name of the resource you want to
+resize the volume for.
+
+1. Make a change to `charts/gsp-system/values.yml` with your desired
+   sizing, like [in PR 343](https://github.com/alphagov/gsp/pull/343):
+
+```yaml
+...
+persistence:
+  worker:
+    size: 128Gi
+```
+
+1. When merged, the PR will be released to sandbox. Changes must pass
+   through sandbox via the full release process to make sure we haven't
+   broken anything before going to the Verify cluster.
+
+1. Pods in StatefulSets each have a volume. For Kubernetes to be able
+   to resize the volumes, we need to detach them from the pods, by
+   destroying the StatefulSet. To destroy the StatefulSet, run:
+
+   ```sh
+   $ gds sandbox kubectl -n gsp-system delete statefulset --cascade=false gsp-concourse-worker
+   ```
+
+   You should destroy the StatefulSet when the Concourse release pipeline starts to fail. You will see an error message similar to:
```

Comment:
It feels to tell someone *when* to run a command after you've already told them to run it.  I can imagine going through this document, running the `delete statefulset` line, and then reading this line and thinking "wait why didn't you tell me before"

Reply:
Fair point. If you move this to before "To destroy the StatefulSet, run:..."  that should be ok hopefully. It pushes the action further away from the start of the numbered step, but you do need the context.